app/assets/stylesheets/_plans.css.scss

```diff
@@ -30,3 +33,41 @@ h2 > .glyphicon-calendar {
 .toggle-section {
   appearance: none;
 }
+
+h2 > .glyphicon-calendar {
+  padding-left: 5px;
+  padding-top: 2px;
+}
+
+.quarter-courses {
+  display: flex;
+  flex-flow: row wrap;
+}
+
+.plan-taken-course {
+  background-color: $gray-lighter;
+}
+
+.plan-course {
+  border: 1px solid #ccc;
+  border-radius: 10px;
+  padding: 15px;
```

Comment:
Properties should be ordered border, border-radius, box-shadow, flex, margin, padding